### PR TITLE
Feature/rit upgrade

### PR DESCRIPTION
**Pull request by viniciussousazup**
**- What I did**
create a upgrade command to download the binaries of last stable version, and add a warning to check if current version is equals to stable version

**- How I did it**
add rit upgrade
add PostRunFunc to check stable version

**- How to verify it**
 run upgrade
 if your version is different of stable version should print a warning

**- Description for the changelog**
rit now have a command to upgrade the binaries to last stable version (rit upgrade) and if your version is not the last stable version a warning is printed after every command.

### CODE REVIEW

File: pkg/cmd/upgrade.go [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+	"github.com/ZupIT/ritchie-cli/pkg/api"
+	"github.com/ZupIT/ritchie-cli/pkg/prompt"
+	"github.com/ZupIT/ritchie-cli/pkg/version/versionutil"
+)
+
+const (
+	upgradeUrlFormat = "https://commons-repo.ritchiecli.io/%s/%s/%s/rit"
+)
+
+type UpgradeUtil interface {
```
Comment (marcosgmgm) [inline pkg/cmd/upgrade.go:22]: separate interface in other file

File: pkg/cmd/upgrade.go [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+func (u DefaultUpgradeUtil) Apply(reader io.Reader, opts update.Options) error {
+	return update.Apply(reader, opts)
+}
+
+type UpgradeCmd struct {
+	upgradeUrl  string
+	upgradeUtil UpgradeUtil
+}
+
+func GetUpgradeUrl(edition api.Edition, resolver versionutil.Resolver) string {
```
Comment (marcosgmgm) [inline pkg/cmd/upgrade.go:37]: Remove Get of func name

File: pkg/file/fileutil/file_util_service.go [hunk 3]
```diff
@@ -0,0 +1,16 @@
+package fileutil
```
Comment (marcosgmgm) [inline pkg/file/fileutil/file_util_service.go:1]: use pkg/file/fileutil/file_util.go

File: pkg/version/versionutil/version.go [hunk 4]
```diff
@@ -0,0 +1,103 @@
+package versionutil
```
Comment (marcosgmgm) [inline pkg/version/versionutil/version.go:1]: change folder to pkg/version

File: pkg/version/versionutil/version.go [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+	FileUtilService  fileutil.FileUtilService
+	HttpClient       *http.Client
+}
+
+type stableVersionCache struct {
+	StableVersion string `json:"stableVersion"`
+	ExpiresAt     int64  `json:"expiresAt"`
+}
+
+func (r DefaultVersionResolver) GetCurrentVersion() (string, error) {
```
Comment (marcosgmgm) [inline pkg/version/versionutil/version.go:41]: Remove Get of func name and separated interface in other file

Comment (viniciussousazup) [reply pkg/version/versionutil/version.go:36]: done : interface separated to other file
to do : Remove Get of func name

File: pkg/version/versionutil/version.go (hunk 4 above)
Comment (viniciussousazup) [reply pkg/version/versionutil/version.go:1]: changed to version_util

File: pkg/cmd/upgrade.go (hunk 1 above)
Comment (viniciussousazup) [reply pkg/cmd/upgrade.go]: create file upgrade on pkg/upgrade

**Pull request merged**